common/lib/xmodule/xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -622,6 +622,49 @@
                 expect(Caption.hideSubtitlesEl).toBeHidden();
             });
 
+            msg = 'on error: attempt fetch transcript with youtubeId if there are ' +
```

Comment:
Can we mention html5 in spec name. It will make the name more explicit between youtube & html5.